docs/guide/configuration/logging.md

```diff
@@ -12,6 +12,9 @@ The log-level can be adjusted at runtime, see [MQTT Topics and Messages](../usag
 advanced:
   # Optional: Logging level, options: debug, info, warn, error (default: info)
   log_level: info
+  # Optional: Configure (override) the logging level per namespace (default: {})
```

Comment:
`Optional: Configure (override) the logging level for specific namespaces (default: {})`

Reply:
Done!